src/static/css/variables.css

```diff
@@ -2,28 +2,52 @@
     /* Colors */
     --color-blue-100-hsl: 234, 100%, 96%;
     --color-blue-100: hsl(var(--color-blue-100-hsl));
-    --color-blue-500-hsl: 234, 62%, 51%;
-    --color-blue-500: hsl(var(--color-blue-500-hsl));
-    --color-blue-700-hsl: 234, 61%, 33%;
+    --color-blue-600-hsl: 234, 62%, 51%;
+    --color-blue-600: hsl(var(--color-blue-600-hsl));
+    --color-blue-700-hsl: 234, 72%, 39%;
     --color-blue-700: hsl(var(--color-blue-700-hsl));
-    --color-green-100-hsl: 170, 37%, 97%;
+    --color-green-100-hsl: 170, 38%, 97%;
     --color-green-100: hsl(var(--color-green-100-hsl));
-    --color-green-300-hsl: 170, 54%, 89%;
-    --color-green-300: hsl(var(--color-green-300-hsl));
-    --color-green-500-hsl: 175, 71%, 69%;
+    --color-green-200-hsl: 170, 54%, 89%;
```

Comment:
Interestingly this hsl value correspond to #d4f2ed, but when I convert the hex value on figma (#d3f2ed), I get this hsl value. I guess there's precision issue for the conversion logic.

Reply:
Oh interesting. I think we should stick with the Figma converter, I didn't find it until today.